server/index.js

```diff
@@ -31,72 +32,62 @@ app.get('/availableAt/:itemId/', (req, res) => {
     .catch((err) => {
       res.status(500).send(err);
     });
-});
 
+});
 
-// //Create
-// app.post('/availableAt', (req, res) => {
-//   console.log('POST received!');
-//   CrudOps.createItemAvailability(req.body.data)
-//     .then((data) => {
-//       res.status(200).send(data);
-//     })
-//     .catch((err) => {
+app.post('/availableAt', (req, res) => {
+  let data = req.body;
+  console.log('POST received! Data: ', data);
 
-//       res.sendStatus(400);
-//     });
-//   //   , (err, data) => {
-//   //   if (err) {
-//   //     res.sendStatus(400);
-//   //   } else {
-//   //     res.status(200).send(data);
-//   //   }
-//   // }
-//   // );
-// });
+  addData(data)
+    .then((data) => {
+      if (data.ok) {
+        res.status(200).send(data);
+      } else {
+        res.sendStatus(500);
+      }
+    })
+    .catch((err) => {
+      res.status(500).send(err);
+    });
+});
 
-// //Read
-// app.get('/availableAt/:itemId/', (req, res) => {
-//   console.log('GET received! ID: ', req.params.itemId);
-//   CrudOps.readItemAvailability(req.params.itemId)
-//     .then((data) => {
-//       // console.log('data: ', data);
-//       if (Object.keys(data).length === 0) {
-//         res.sendStatus(404);
-//       } else {
-//         res.status(200).send(data);
-//       }
-//     })
-//     .catch((err) => {
-//       res.status(500).send(err);
-//     });
-// });
+app.put('/availableAt/:itemId', (req, res) => {
+  let itemId = req.params.itemId;
+  let data = req.body;
+  console.log('Put received! ID: ', itemId);
 
-// //Update
-// app.put('/availableAt/:itemId/', (req, res) => {
-//   console.log('PUT received! ID: ', req.params.itemId);
-//   CrudOps.updateItemAvailability(req.params.itemId, { ...req.body.data })
-//     .then((data) => {
-//       res.status(200).send(data);
-//     })
-//     .catch((err) => {
-//       res.status(500).send(err)
-//     });
-// });
+  updateData(itemId, data)
+    .then((data) => {
+      if (data.ok) {
```

Comment:
data.ok? What does this mean?

Reply:
The data shape for the response for updates in couch looks like this:
`{
    "id": "blah",
    "ok": true,
    "rev": "2-056f5f44046ecafc08a2bc2b9c229e20"
}`